src/components/Header/index.jsx

```diff
@@ -39,28 +39,27 @@ const GordonHeader = ({ onDrawerToggle }) => {
    * The checks use regular expressions to check for matches in the url.
    */
   const updateTabHighlight = () => {
-    let currentPath = window.location.pathname;
     // Tab url regular expressions must be listed in the same order as the tabs, since the
     // indices of the elements in the array on the next line are mapped to the indices of the tabs
-    let urls = [
+    const urls = [
       /^\/$/,
       /^\/involvements\/?$|^\/activity/,
       /^\/events\/?$/,
-      /^\/people$/,
+      /^\/people$|^\/myprofile|^\/profile/,
       /^\/timesheets$/,
     ];
-    setTabIndex(false);
+    let currentPath = useLocation().pathname;
```

Comment:
The `useLocation` function provided by React Router is a Hook. As I mentioned in a different review, Hooks have to obey certain rules in React in order for everything to work properly, which are detailed in the [React Docs](https://react.dev/warnings/invalid-hook-call-warning).

The relevant one here is "always use Hooks at the top level of your React function", which specifically includes "Don’t call Hooks nested functions". The `updateTabHighlight` function is one such nested function, and is not at the "top level" of the function, so this call to `useLocation` violates that rule.

It works okay in this case, but there are many subtle ways that this code could change which would break React, so we shouldn't leave it as is.

Fixing this is actually really simple - just move the invocation of `useLocation` outside of `updateTabHighlight`. For Example:
```javascript
const location = useLocation();
...
const updateTabHighlight = () => {
  ...
  const currentPath = location.pathname;
  ...
};
```

That will fix the most severe issue that we're calling a hook at not the top level.